app/src/main/java/ru/suslovalex/androidacademyapp/adapters/AdapterMovies.kt

```diff
@@ -44,8 +44,9 @@ class MovieViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
     private val timeLimit: TextView = itemView.findViewById(R.id.time_limit)
 
     fun bind(movie: Movie) {
+        itemView.transitionName = (R.string.transition+position).toString()
```

Comment:
@AleksandrAlex Are you sure that `R.string.transition` will return you the `String`, it is `Int` value. You need to use `getString(R.string.transition)` and add a position in Kotlin style